File: 3/3.1.swift
import Foundation

//後ほど説明 (*1)
enum Currency {
    case yen, dollar
}

//[BAD]
//そもそもビルドができないようにされている
//パラメータを初期化しないといけない
class Money {
    var amount: Int
    var currency: Currency
}

let money = Money()

File: 3/3.1.dart
//後ほど説明 (*1)
enum Currency { yen, dollar }

//[BAD]
//そもそもビルドができないようにされている
//パラメータを初期化しないといけない
class Money {
  int amount;
  Currency currency;
}

void main() {
  final money = Money();
}
